src/strands/tools/executors/_executor.py

```diff
@@ -226,6 +226,7 @@ def _execute(
         cycle_trace: Trace,
         cycle_span: Any,
         invocation_state: dict[str, Any],
+        **kwargs: Any,
```

Comment:
@zastrowm @Unshure The goal over here is to pass the `structured_output_context` to the tool executor `_execute`.
I'm curious to get your thoughts if you think it's better to use `kwargs` (like i'm doing here)  or is it better to make it an explicit param?